test/unit/sb3.js

```diff
@@ -270,16 +274,80 @@ test('complex (binary)', t => {
 
 test('extensions', t => {
     analysis(extensionsBinary, (err, result) => {
-        t.true(typeof err === 'undefined' || err === null);
+        t.ok(typeof err === 'undefined' || err === null);
         t.type(result, 'object');
 
         t.type(result.extensions, 'object');
         t.equal(result.extensions.count, 2);
-        t.deepEqual(result.extensions.id, [
+        t.same(result.extensions.id, [
             'translate',
             'text2speech'
         ]);
 
         t.end();
     });
 });
+
+test('regression test IBE-198', t => {
```

Comment:
We might forget what this means - I wonder if there's a way we could articulate what it's supposed to do in just a few words so that if it ever fails we'd know exactly why (basically "ensure extension list is typechecked", right)?

Reply:
Yeah, makes sense, I'll update it and then merge